### PR TITLE
Add error templates

### PR DESCRIPTION
**Pull request by aliuk2012**
Adds error page templates.

These use GOV.UK Frontend but apart from that the content is identical to the existing templates in digitalmarketplace-frontend-toolkit.

Will help to reduce the number of files in this PR.

https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/560

Ticket: https://trello.com/c/pT5lRLOG

### CODE REVIEW

**Comment by lfdebrux** [comment]
Nice, I like that we're putting our stuff in `package/digitalmarketplace`

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 package/govuk-frontend/
+package/digitalmarketplace/
```
Comment (lfdebrux) [inline .gitignore]: I noticed that govuk-frontend does commit the package directory.

What if we didn't add this line and also removed the line above ignoring `package/govuk-frontend/`? Then we wouldn't need https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/13

File: tasks/gulp/build.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
 const { task, src, dest } = require('gulp')
-const debug = require('gulp-debug')
```
Comment (lfdebrux) [inline tasks/gulp/build.js]: If we're removing this we can also remove `gulp-debug` from `package.json`? Might want to do as a separate commit.

**Comment by lfdebrux** [comment]
In govuk-frontend, templates are split into different directories depending on how they should be used (see [`app/views`](https://github.com/alphagov/govuk-frontend/tree/v2.13.0/app/views); so pages like these would go into a folder `layouts`. Is this folder structure something that would be useful for us to copy?

**Comment by aliuk2012** [comment]
> In govuk-frontend, templates are split into different directories depending on how they should be used (see [`app/views`](https://github.com/alphagov/govuk-frontend/tree/v2.13.0/app/views) [](/alphagov/govuk-frontend/tree/HEAD@{2019-10-18T11:28:42Z}/app/views)); so pages like these would go into a folder `layouts`. Is this folder structure something that would be useful for us to copy?

I did consider it but I think it might break other things and require further work eg. changes like to digitalmarketplace-utils error method.

I am trying to follow [this folder structure](https://github.com/alphagov/govuk-frontend/tree/master/package/govuk).

File: tasks/gulp/build.js (hunk 2 above)
Comment (aliuk2012) [reply tasks/gulp/build.js]: Good spot, I've removed gulp-debug

File: .gitignore (hunk 1 above)
Comment (aliuk2012) [reply .gitignore]: As discussed offline, we do not want to version compiled code only src code.

File: gulpfile.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 // Clean compiled files/folders
```
Comment (lfdebrux) [inline gulpfile.js]: This comment might need updating?

**Comment by lfdebrux** [comment]
All looks good to me! 👍
